### PR TITLE
py: non-regression of user-feedback flow

### PR DESCRIPTION
**Pull request by tarilabs**
## Description
edit: this add a non-regression test based on user feedback

## How Has This Been Tested?
`make test-e2e`

## Merge criteria:

- All the commits have been [_signed-off_](https://github.com/kubeflow/community/tree/master/dco-signoff-hook#signing-off-commits)  (To pass the `DCO` check)

- [x] The commits have meaningful messages; the author will squash them after approval or in case of manual merges will ask to merge with squash.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work.
- [x] Code changes follow the [kubeflow contribution guidelines](https://www.kubeflow.org/docs/about/contributing/).

If you have UI changes

- [ ] The developer has added tests or explained why testing cannot be added.
- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Verify that UI/UX changes conform the UX guidelines for Kubeflow.

### CODE REVIEW

**Comment by tarilabs** [comment]
okay it's a bit weird this passes on CI 🤔 😅

**Comment by tarilabs** [comment]
I _think_ the local failure I was experiencing is due to the fact I got both docker (non-desktop, just docker) and Podman (and Podman Desktop) installed.

**Comment by tarilabs** [comment]
> I _think_ the local failure I was experiencing is due to the fact I got both docker (non-desktop, just docker) and Podman (and Podman Desktop) installed.

confirmed, TIL

image

File: clients/python/tests/test_client.py [hunk 1]
```diff
@@ -230,6 +230,39 @@ async def test_update_preserves_model_info(client: ModelRegistry):
     assert updated_ma.model_format_version == model_fmt_version
 
 
+@pytest.mark.e2e
+async def test_update_model_artifact_uri_reuse(client: ModelRegistry):
```
Comment (isinyaaa) [inline clients/python/tests/test_client.py:234]: Nit: this test might be better named as 
```suggestion
async def test_update_existing_model_artifact(client: ModelRegistry):
```
as __reuse__ isn't too clear about what's being done with it.

**Review by rareddy — APPROVED** [review]
/lgtm
/approve

**Pull request merged**